SeaPublicWebsite/Views/EnergyEfficiency/ActionPlan/Partials/_ActionPlanBeforeMainLinks.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using SeaPublicWebsite.BusinessLogic.Models.Enums
+@using SeaPublicWebsite.Models.EnergyEfficiency
+@model SeaPublicWebsite.Models.EnergyEfficiency.ActionPlanViewModel;
+
+@if (!Model.IsPdf)
```

Comment:
I find it a bit weird that we've got a view that's sometimes empty. I think it would make more sense to keep the `IsPdf` check on the main view rather than this partial.
That means we're repeating ourselves a bit but I prefer to know that if I see this partial being used on a page then its contents will actually show up.